docs/changes/coordinates/11676.feature.rst

```diff
@@ -0,0 +1,3 @@
+Adds new ``HADec`` built-in frame with transformations to/from ``ICRS`` and ``CIRS``.
+This frame complements ``AltAz`` to give observed coordinates 
+in the ``ITRS`` for an equatorially mounted telescope.
```

Comment:
can we clarify this what's new entry for users without more coordinates knowledge? Perhaps "...complements ``AltAz`` to give observed coordinates (hour angle and declination) in the ``ITRS`` for an equatorially mounted telescope."?

Reply:
Sure, no problem. Thanks for the review.